src/index.html

```diff
@@ -30,10 +30,11 @@ <h2>Education</h2>
           <!-- Modal content -->
           <div class="modal-content">
             <span class="close">&times;</span>
-            <ol  id="modalInformation"></ol>
+            <p  id="modalInformation"></p>
+            <a id="linkToProject" href="https://github.com/JLMartinez1980"></a>
```

Comment:
This is nitpicking, but this id is _actually_ a link to your github. Should probably be `id="linkToGithub`.